README.md

```diff
@@ -1,13 +1,13 @@
-# Your Project Name
+# Simulation Intelligente d'un Système Feu de Passage Piéton
 
```

Comment:
Project is too simple and does not meet requirements. You need to use at least 3 labs and 3 peripherals for your project. For example you can add a proximity sensor or an rfid sensor and a buzzer for your project which will help visually impaired people traverse the road.